08_deploy_bmo.sh

```diff
@@ -13,8 +13,23 @@ cp -r $BMOPATH/deploy ocp/.
 sed -i 's/namespace: .*/namespace: openshift-machine-api/g' ocp/deploy/role_binding.yaml
 
 # Start deploying on the new cluster
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/service_account.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role_binding.yaml
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/operator.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+oc --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/service_account.yaml
+oc --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role.yaml
+oc --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role_binding.yaml
+oc --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+oc --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/operator.yaml
+
+# Give the operator a chance to quiesce. Ideas to check instead of sleep?
+sleep 10
+
+oc --config ocp/auth/kubeconfig proxy &
+proxy_pid=$!
+
+for node in $(oc --config ocp/auth/kubeconfig get nodes -o template --template='{{range .items}}{{.metadata.uid}}:{{.metadata.name}}{{"\n"}}{{end}}'); do
```

Comment:
Can you leave a comment in here that this is a hack related to #260 where we're tracking getting this done automatically?